src/api/SOOSHooksApiClient.ts

```diff
@@ -0,0 +1,61 @@
+import { AxiosInstance } from "axios";
+import { SOOS_CONSTANTS } from "../constants";
+import SOOSApiClient from "./SOOSApiClient";
+
+interface IContributorAuditMetadata {
+  scriptVersion: string;
+  days: number;
+}
+
+interface IContributorAuditModel {
+  metadata: IContributorAuditMetadata;
+  organizationName: string;
+  contributors: IContributorAuditRepositories[];
+}
+
+interface IContributorAuditRepositories {
+  username: string;
+  repositories: IContributorAuditRepository[];
+}
+
+interface IContributorAuditRepository {
+  id: number;
+  name: string;
+  lastCommit: string;
+  isPrivate: boolean;
+}
+
+class SOOSHooksApiClient {
+  private readonly baseUri: string;
+  private readonly apiKey: string;
+  private readonly client: AxiosInstance;
+
+  constructor(apiKey: string, baseUri: string = SOOS_CONSTANTS.Urls.API.Hooks) {
+    this.apiKey = apiKey;
+    this.baseUri =
+      baseUri != SOOS_CONSTANTS.Urls.API.Hooks
+        ? baseUri.replace("api.", "api-hooks.")
+        : SOOS_CONSTANTS.Urls.API.Hooks; // TODO - REMOVE THIS LOGIC FROM HERE, maybe put in analysis service
```

Comment:
yes, add this to the analysis service - and for the scm service do the same create pattern as the analysis service